src/command-parser.js

```diff
@@ -499,26 +499,26 @@ export function parseSEARCH (response) {
 
 /**
  * Parses COPY and UID COPY response.
+ * https://tools.ietf.org/html/rfc4315
  * @param {Object} response
  * @returns {{destSeqSet: string, srcSeqSet: string}} Source and
- * destination uid sequence sets.
+ * destination uid sets if available, undefined if not.
  */
 export function parseCOPY (response) {
-  const value = { srcSeqSet: '', destSeqSet: '' }
   const copyuid = response && response.copyuid
-
   if (copyuid) {
-    value.srcSeqSet = copyuid[1]
-    value.destSeqSet = copyuid[2]
+    return {
+      srcSeqSet: copyuid[1],
+      destSeqSet: copyuid[2]
+    }
```

Comment:
Out of curiosity, what is in copyuid[0]?

Reply:
These are UIDVALIDITY numbers/codes, afaict, https://tools.ietf.org/html/rfc3501#section-7.1